src/main/java/com/github/tomakehurst/wiremock/matching/WeightedAggregateMatchResult.java

```diff
@@ -31,6 +31,12 @@ public class WeightedAggregateMatchResult extends MatchResult {
   private final Lazy<Pair<Boolean, List<SubEvent>>> resultAndEvents;
 
   public WeightedAggregateMatchResult(List<WeightedMatchResult> matchResults) {
+    this(matchResults, null);
+  }
+
+  public WeightedAggregateMatchResult(
+      List<WeightedMatchResult> matchResults, DiffDescription diffDescription) {
```

Comment:
I'm wondering if we should aggregate all the diff descriptions somehow rather than passing a single one in, as I'm not sure how we'd assign a single diff description to an aggregate. Possibly warrants some further discussion.

Reply:
Maybe we can get away with returning null here for now since aggregate results are never used in diff reports.